app/models/person.rb

```diff
@@ -17,9 +17,9 @@ class Person < ApplicationRecord
       person.diaspora_handle
     }, :as => :diaspora_id
     t.add lambda { |person|
-      {:small => person.profile.image_url(:thumb_small),
-       :medium => person.profile.image_url(:thumb_medium),
-       :large => person.profile.image_url(:thumb_large) }
+      {:small => person.profile.image_url(size: :thumb_small),
```

Comment:
- Layout/AlignHash: Align the elements of a hash literal if they span more than one line.
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax.